src/constants.js

```diff
@@ -280,3 +280,9 @@ export const FEATURE_TOGGLE_DEFAULTS = {
 export const colors = {
   purpleHathor: scssColors.purpleHathor,
 }
+
+/**
+ * Number of elements in the list of addresses
+ * when changing the address of a nano contract
+ */
+export const CHANGE_ADDRESS_LIST_COUNT = 5;
```

Comment:
suggestion: This name feels confusing to me, especially in the global context as it is. _Change address_ is, until now, a concept related to outputs returning tokens back to the sender after partially consuming an utxo.

What do you think about `NANO_ADDRESS_CHANGE_LIST_COUNT`?

Reply:
Removed the `CHANGE` from the variable name. Changed to `NANO_UPDATE_ADDRESS_LIST_COUNT`. What do you think? https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/3a28ae1d2ab7d56576bd140954264140c367f0f7